special-use/special-use-suds-mapping.json

```diff
@@ -6,97 +6,97 @@
       "applicant-info": {
         "type": "object",
         "properties": {
-      		"firstName": { "type": "string", "suds-application": "Contacts", "suds-field": "First Name"},
-      		"lastName": {	"type": "string", "suds-application": "Contact", "suds-field": "Last Name" },
+      		"firstName": { "type": "string", "basic-api-field" : "FIRST_NAME"},
+      		"lastName": {	"type": "string", "basic-api-field" : "LAST_NAME" },
           "dayPhone": { "$ref": "#/definitions/phone-number" },
           "eveningPhone": { "$ref": "#/definitions/phone-number" },
-          "emailAddress": { "type": "string", "suds-application": "Contact", "suds-field": "E-mail"},
-          "mailingAddress": { "type": "string", "suds-application": "Contact", "suds-field": "Address 1"},
-          "mailingAddress2": { "type": "string", "suds-application": "Contact", "suds-field": "Address 2"},
-          "mailingCity": { "type": "string", "suds-application": "Contact", "suds-field": "City"},
-          "mailingState": { "type": "string", "suds-application": "Contact", "suds-field": "State/Province" },
-          "mailingZIP": { "type": "integer", "suds-application": "Contact",  "suds-field": "Postal code" },
+          "emailAddress": { "type": "string", "basic-api-field" : "EMAIL"},
+          "mailingAddress": { "type": "string", "basic-api-field" : "ADDRESS_1"},
+          "mailingAddress2": { "type": "string", "basic-api-field" : "ADDRESS_2"},
```

Comment:
Ah yes. Should we ask Xin to fix on the basic API side? It seems kind of awkward for us to say "if empty submit a space character."